cmd/progressbar/progressbar.go

```diff
@@ -138,11 +158,17 @@ func (pb *progressBar) handleDynamicBarUpdate(data DynamicBarUpdate) {
 		log.Panicf("trying to update a bar that doesn't exist: %s (%v)", displayName, data)
 	}
 
+	currentTotal := val.Current()
+
 	if data.TotalChange > 0 {
-		val.SetTotal(val.Current()+data.TotalChange, false)
+		val.SetTotal(currentTotal+data.TotalChange, false)
```

Comment:
I'm confused because TotalChange is never set now

Reply:
see above comment

Reply:
so please just add a comment to explain what's going on here because it's unclear how it works.
e.g. we had `1, 4` before (a gap between current and total), and it seems that you worked that around some other way. so please explain